modules/ROOT/partials/toolbar-button-ids/mergetags-toolbar-buttons.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,3",options="header"]
+|===
+|Toolbar button identifier |Description
+|`+mergetags+` |Inserts a merge tag from a nested menu. The nested menu contains the merge tags list, as specified in xref:#mergetags_list[`+mergetags_list+`]. The toolbar button includes a search feature for finding specific merge tags.
```

Comment:
This xref won't work when included on multiple pages which is will need to be. In general it should only ever be relative like this when the xref target is on the current page.

Reply:
fixed